compilers/src/make.js

```diff
@@ -2,8 +2,8 @@ const fs = require('fs')
 
 const domains = fs.readFileSync('../../domains', 'utf8').split('\n')
 
-fs.appendFile('hostfile', '##COINHIVE DOMAINS\n')
+fs.appendFile('../../hostfile', '##COINHIVE DOMAINS\n', ()=>{})
```

Comment:
This is showing inconsistencies on my machine. sometimes one or two domains are prepended in before of the title.